stdlib/public/core/StringObject.swift

```diff
@@ -856,6 +856,14 @@ extension _StringObject {
     return Builtin.reinterpretCast(largeAddressBits)
 #endif
   }
+
+  @_alwaysEmitIntoClient
+  @inlinable
+  @inline(__always)
+  internal var owner: AnyObject? {
+    guard self.isMortal else { return nil }
+    return Builtin.reinterpretCast(largeAddressBits)
```

Comment:
@Catfish-Man this would be another place where we expect to not use the large nibble